src/Microsoft.CodeAnalysis.Razor.Workspaces/Host/IDocumentExcerptService.cs

```diff
@@ -1,79 +0,0 @@
-// Copyright (c) Microsoft.  All Rights Reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-#if DOCUMENT_SERVICE_FACTORY
-
-using System.Collections.Immutable;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.Classification;
-using Microsoft.CodeAnalysis.Text;
-
-namespace Microsoft.CodeAnalysis.Host
-{
-    /// <summary>
-    /// excerpt some part of <see cref="Document"/>
-    /// </summary>
-    internal interface IDocumentExcerptService : IDocumentService
```

Comment:
Just verifying, all these bits are now done on the managed project system side right?

Reply:
Everything in `Microsoft.CodeAnalysis.Host` is Roslyn. 

The changes I removed from `XyzProjectHost` are just old workaround cruft that isn't needed.

the Managed project system tells Roslyn about `.cshtml` files that get added, and they call through to us through `IDynamicFileInfoProvider`